flight/flight.py

```diff
@@ -0,0 +1,101 @@
+import asyncio
+import threading
+from mavsdk import System
+import mavsdk as sdk
+from .states import STATES, State
+
+
+class Comm:
+    state: str = "start"
+
+
+class StateMachine:
```

Comment:
Is this a temporary replacement for states/State.py? or something new?

Reply:
Thats the state machine, then the states are separate